synapse/lib/aha.py

```diff
@@ -177,6 +177,23 @@ async def modAhaSvcInfo(self, name, svcinfo):
 
         return await self.cell.modAhaSvcInfo(name, svcinfo)
 
+    async def getAhaSvcMirrors(self, name):
+        '''
+        Return list of AHA svcinfo dictionaries for mirrors of a service.
+        '''
+        svcinfo = await self.cell.getAhaSvc(name)
+        if svcinfo is None:
+            return None
```

Comment:
should  we return a empty list? that way we don't have type confusion on the return value.

Reply:
I went with the same pattern as the getAhaSvc API since we would want to differentiate between "service is configured and has no mirrors" and "service is not configured".  The stormlib and cell APIs raise for the None case, so those only ever return lists.  Could possibly raise here for the AHA API too?